supervision/detection/core.py

```diff
@@ -178,6 +178,10 @@ def from_yolov5(cls, yolov5_results) -> Detections:
             ```
         """
         yolov5_detections_predictions = yolov5_results.pred[0].cpu().cpu().numpy()
+
+        if np.asarray(yolov5_detections_predictions[:, :4]).shape[0] == 0:
```

Comment:
Tested and not needed.

Reply:
.numpy() ones not needed okay. I will revert those but leave others then.
```suggestion
        if np.asarray(yolov5_detections_predictions[:, :4]).shape[0] == 0:
```